fields/types/cloudinaryimage/CloudinaryImageType.js

```diff
@@ -277,6 +277,21 @@ cloudinaryimage.prototype.getData = function (item) {
 	return typeof value === 'object' ? value : {};
 };
 
+cloudinaryimage.prototype._originalGetOptions = cloudinaryimage.prototype.getOptions;
+
+cloudinaryimage.prototype.getOptions = function () {
+	this._originalGetOptions();
+	// We are performing the check here, so that if cloudinary secure is added
+	// to keystone after the model is registered, it will still be respected.
+	// Setting secure overrides default `cloudinary secure`
```

Comment:
You've got me curious. What's the use case for setting `secure`/`cloudinary secure` after the models have been registered?

Reply:
It's mostly about the ordering of your `keystone.js` file. It is possible and valid to import your models and then after that line set `cloudinary secure`, however rather obtusely without checking again, you can end up not applying the option it looks like you set.

Reply:
I see. Thanks for the explanation.